src/groundlight/experimental_api.py

```diff
@@ -1046,3 +1048,68 @@ def get_detector_metrics(self, detector: Union[str, Detector]) -> dict:
             detector = detector.id
         obj = self.detectors_api.get_detector_metrics(detector)
         return obj.to_dict()
+
+    def get_raw_headers(self) -> dict:
+        """
+        Get the raw headers for the current API client
+
+        :return: a dictionary containing the raw headers
+        """
+        headers = {}
+        # see generated/groundlight_openapi_client/api_client.py update_params_for_auth
+        headers["x-api-token"] = self.api_client.configuration.api_key["ApiToken"]
+        # We generate a unique request ID client-side for each request
+        headers["X-Request-Id"] = _generate_request_id()
+        headers["User-Agent"] = self.api_client.default_headers["User-Agent"]
+        headers["Accept"] = "application/json"
+        return headers
+
+    def make_raw_rest_request(self, method: str, endpoint: str, body: Union[dict, None] = None) -> dict:
+        """
+        Make a raw REST request to the specified URL
+
+        :param method: the HTTP method to use
+        :param endpoint: the endpoint to send the request to - the url path appended after the
+            endpoint including a / at the beginging
+        :param body: the request body
+
+        :return: a dictionary containing the raw response
+        """
+        headers = self.get_raw_headers()
+        url = f"{self.api_client.configuration.host}{endpoint}"
+        response = requests.request(method, url, headers=headers, json=body)
+        return response.json()
+
+    def make_generic_api_request(  # noqa: PLR0913 # pylint: disable=too-many-arguments
+        self,
```

Comment:
```suggestion
        self,
        *,
```

We could require the arguments to be named, not positionally-ordered.  Feels like a good practice for these methods with tons of arguments, although this one is mercifully more succinct than the openapi provided ones.

Is this the right syntax for that?